components/automate-ui/src/app/pages/+compliance/+credentials/components/credentials-list.html

```diff
@@ -8,12 +8,17 @@
     <div class="credentials-list-body">
       <chef-toolbar>
         <app-authorized [anyOf]="['/secrets', 'post']">
-          <chef-button primary [routerLink]="['/settings/node-credentials/add']">
-            Add Credential
-          </chef-button>
+          <div *ngIf="credentialsList.items.length === 0" class="credentials-empty-state">
+            <p>Create the first credential to get started!</p>
+          </div>
+          <div [ngClass]="credentialsList.items.length === 0 ? 'credentials-empty-state' : ''">
+            <chef-button primary [routerLink]="['/settings/node-credentials/add']">
+              Add Credential
+            </chef-button>
+          </div>
         </app-authorized>
       </chef-toolbar>
-      <chef-table class="credentials-list-table" (sort-toggled)="onSortToggled($event)">
+      <chef-table *ngIf="credentialsList.items.length > 0" class="credentials-list-table" (sort-toggled)="onSortToggled($event)">
```

Comment:
Me again. 😁  Looks like this one is missing an `<app-authorized>` around the table, too. Let me know if you are unsure of how to determine the arguments.

Reply:
@msorens I trying with the  `<app-authorized>` but it is not worked, means the table is not showing after adding the node credentials.

Reply:
Your arguments should be '/secrets/search' and 'post'.

Reply:
I added the changes and updated the PR.